src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioDiagnosticListTable.BuildTableDataSource.cs

```diff
@@ -229,6 +229,9 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         case ProjectGuids:
                             content = item.ProjectGuids;
                             return ((Guid[])content).Length > 0;
+                        case SuppressionStateColumnDefinition.ColumnName:
```

Comment:
how build diagnostic know whether it is suppressed or not? is out of proc build now log suppressed errors as well? I dont see diagnostic created from out of proc build ever set IsSuppressed.

are you trying to always say "Active" for diagnostic from build?